### PR TITLE
Update ACLType in index.d.ts@types/ali-oss

### PR DESCRIPTION
**Pull request by imhele**
### From
```ts
type ACLType = "public-read-write" | "public-read" | "and private";
```
### To
```ts
type ACLType = "public-read-write" | "public-read" | "private";
```

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [@types/ali-oss - index.d.ts#L32](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ali-oss/index.d.ts#L32)

### CODE REVIEW

**Review by ptrdu — APPROVED** [review]
Accept

**Review by ptrdu — COMMENTED** [review]
[Ali-oss Source code: From Line 90 to Line 100](https://github.com/ali-sdk/ali-oss/blob/35c729fd225352128f8c712aebd2875d8d9d35e4/lib/object.js#L90)

Form the source code, i cannot find the headers property in the returned ret object, are you sure there is a headers property in the PutObjectResult type?

**Review by ptrdu — APPROVED** [review]

**Comment by imhele** [comment]
@ptrdu At https://github.com/ali-sdk/ali-oss/blob/master/lib/object.js#L113

File: types/ali-oss/index.d.ts [hunk 1]
```diff
@@ -180,6 +181,7 @@ declare namespace OSS {
         name: string;
         data: object;
         res: NormalSuccessResponse;
+        headers?: object;
```
Comment (ptrdu) [inline types/ali-oss/index.d.ts]: This line @imhele

Comment (imhele) [reply types/ali-oss/index.d.ts]: https://github.com/ali-sdk/ali-oss/blob/master/lib/object.js#L76

Comment (ptrdu) [reply types/ali-oss/index.d.ts]: I think you should add the headers type in the `PutObjectOptions ` interface, not the `PutObjectResult ` interface
@imhele

**Review by ptrdu — APPROVED** [review]
Done
Thank you～

File: types/ali-oss/index.d.ts (hunk 1 above)
Comment (imhele) [reply types/ali-oss/index.d.ts]: oh, thanks

**Pull request merged**